x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/apm_metrics.json

```
@@ -0,0 +1,53 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "Detects anomalies in transaction duration, throughput and error percentage for metric data.",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "summary_count_field_name" : "doc_count",
+    "detectors" : [
+      {
+        "detector_description" : "high duration by transaction type for an APM service",
+        "function" : "high_mean",
+        "field_name" : "transaction_duration",
```

`transaction_duration` is probably fine. But if we are renaming the others we might as well improve this too:

```suggestion
        "field_name" : "latency",
```